### ISSUE TITLE
 Library not found for -lprotobuf

### ISSUE THREAD

**Issue by petersyden** [eth-siplab/SVG2Keynote-gui#4]
I can't compile even though protobuf 3.19.4 is installed. I'm on an m1 Mac running 12.1

**Comment by hukl** [comment]
Yeah have the same issue - first the problem was that it couldn't find the homebrew directory which now is located in `/opt/homebrew/lib` but it seems everything is locked to intel architecture so I couldn't really build / run it on the m1 mac.

**Comment by kosuvorov** [comment]
> I can't compile even though protobuf 3.19.4 is installed. I'm on an m1 Mac running 12.1

Have you figured it out?

**Comment by hukl** [comment]
After much hassle I got it to work - or lets say to compile and run but then when I tried to convert an SVG I got a bunch of path errors in the console and after two hours I let it go

**Comment by kosuvorov** [comment]
When I'm trying to build the app in Xcode I see an error message: "Build input file cannot be found". Do you know wow can I solve this?

I have a macbook with m1pro, Mac os 12.2 and the latest xcode version

**Comment by kosuvorov** [comment]
Also I found the alternative plugin - Ir2office. Link - https://www.recosoft.com/products/ir2office/. I tested a trial version and from perspective it works pretty good.
